### PR TITLE
Feature/gt 129 end of the day report

### PR DESCRIPTION
**Pull request by egecnr**
This also fixes the timer on minigame issues.

### CODE REVIEW

**Comment by logoffro1** [comment]
If you didnt complete any tasks, you get a weird int so make sure to solve that
image

Also I dont know if it's from your branch, but the timer is moved too up, outside of the view

**Comment by egecnr** [comment]
Yeah the hierarchy change in the timer was planned. Since it is going to be displayed in more  places, it can be handled outside of the usual canvas. 

Thank you for noticing the bug that requires a one liner fix. I will work on it after the consultancy.

I also wanna work on this again as a refinement feature after vertical slice to make it better visually.

File: Assets/Scripts/HUD/EndOfDayReport.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        Success.text += $"  {playerReportData.GetTheSuccessfulMinigameNumber()}";
+        Fail.text += $"  {playerReportData.GetTheFailedMinigameNumber()}";
+        SliderValue.text += $"  {ProgressBar.Instance.GetSlideValue().ToString("F2")}%";
+        TotalTasknumber.text += $"  {playerReportData.GetTotalTaskNumber()}";
+        Cursor.lockState = CursorLockMode.None;
+        Cursor.visible = true;
+    }
+
+    // Update is called once per frame
+    void Update()
```
Comment (dbecu) [inline Assets/Scripts/HUD/EndOfDayReport.cs:38]: Remove this please

File: Assets/Scripts/MiniGameManager.cs [hunk 2]
```diff
@@ -28,27 +30,22 @@ private void Awake()
...
+        PlayerData = FindObjectOfType<PlayerReportData>();
+    }
 
-        if (Input.GetKeyDown(KeyCode.V))
-        {
-            UIManager.Instance.ChangeCanvasShown();
-            miniGame = Instantiate(tetrisGamePrefab, new Vector3(0, 0, 100), tetrisGamePrefab.transform.rotation);
-            IsPlaying = true;
-        }
+    private void Update()
```
Comment (dbecu) [inline Assets/Scripts/MiniGameManager.cs:38]: Please remove this too

**Review by dbecu — CHANGES_REQUESTED** [review]
Timer countdown is missing

**Comment by egecnr** [comment]
for my version. Timer is there can you try to pull?

File: Assets/Scripts/HUD/EndOfDayReport.cs (hunk 1 above)
Comment (egecnr) [reply Assets/Scripts/HUD/EndOfDayReport.cs:38]: Done

**Comment by egecnr** [comment]
@logoffro1  @dbecu  thank you for the feedback! I implemented everything could you give it another shot after pulling?

**Pull request merged**